dashboard/lib/sqs/queue_processor_config.rb

```diff
@@ -4,42 +4,33 @@ module SQS
 
   # Immutable configuration for a queue processor.
   class QueueProcessorConfig
-    attr_reader :queue_uri, :handler, :num_workers_per_processor, :num_processors
+    attr_reader :queue_url, :handler, :num_workers_per_processor, :num_processors, :logger
     attr_reader :initial_max_rate, :dcdo_max_rate_key
-    attr_reader :max_wait_time, :max_batch_size
 
-    # @param [String] queue_uri URI to the SQS queue.
+    # @param [String] queue_url URI to the SQS queue.
     # @param [Integer] initial_max_rate Initial max rate (in messages per second) or 0 for no limit.
     # @param [String] dcdo_max_rate_key DCDO key for the dynamic max rate.
```

Comment:
(I see this is initially from #5530, but this is the first time I've looked at this code) my preference would be to pass a Proc argument (returning a max-rate number) instead of a DCDO key, to keep the queue-processing system decoupled from specifics of the DCDO api.